forge/lib/poolmanager.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+import time
+import multiprocessing
+import signal
+import ConfigParser
+
+from forge.lib.logs import getLogger
+from forge.lib.helpers import timestamp
+
+NUMBER_POOL_PROCESSES = multiprocessing.cpu_count()
+
+dbConfig = ConfigParser.RawConfigParser()
+dbConfig.read('database.cfg')
+logger = getLogger(dbConfig, __name__, suffix=timestamp())
+
+
+# Assure that sub processes don't get keyborad interrupts
+def initProcess():
+    logger.info('Starting process id: %s' % multiprocessing.current_process().pid)
+    signal.signal(signal.SIGINT, signal.SIG_IGN)
+
+
+class PoolManager:
+
+    def __init__(self):
+        self._pool = multiprocessing.Pool(NUMBER_POOL_PROCESSES, initProcess)
+
+    def _abort(self):
+        self._pool.terminate()
+        self._pool.join()
+
+    def numOfProcesses(self):
+        return NUMBER_POOL_PROCESSES
```

Comment:
numOfProcesses, could be intitialized in the __init__ and could take per default the number of CPU or a custom nb of pool process. Minor though